config/initializers/10-patches.rb

```diff
@@ -98,6 +77,12 @@ def full_message(attribute, message)
 
 module ActionView
   module Helpers
+    module Tags
+      Base.class_eval do
+        attr_reader :method_name
```

Comment:
@ulferts I'd rather leave this workaround in. The accessible errors code (e.g. `wrap_with_error_span`) is already a mess at the moment:

  1. this accessible errors code should live somewhere other than a patches initializer (and tested)
  2. fields with errors don't even look right: the HTML structure generated is probably wrong, and pending proper styling (see [User Story #19004](https://community.openproject.org/work_packages/19004) – I think @floriank wanted to take this over)